src/lib/config/invenioConfig.js

```diff
@@ -211,6 +211,10 @@ export const invenioConfig = {
       ill_item_type: 'ill_item_type',
       ill_payment_mode: 'ill_payment_mode',
     },
+    docReq: {
```

Comment:
For consistency use `documentRequests` instead of `docReq`, in general we try to avoid different abbreviations.